CHANGELOG.md

```diff
@@ -3,6 +3,11 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [5.2.3] - TBD
```

Comment:
Let's go with 5.3.0 as this change isn't a bug fix and one could argue that making some classes available downstream is a feature for developers.

Reply:
Also, once this is merged can you raise a similar PR for the 6.x series against master so we keep the changes in sync?

Reply:
yes 👍